app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx

```diff
@@ -88,6 +91,15 @@ export class SimpleWidget extends Component {
     }).isRequired,
     activeDashboardId: PropTypes.number.isRequired,
     isAnalyticsEnabled: PropTypes.bool.isRequired,
+    baseEventParameters: PropTypes.shape({
+      instanceId: PropTypes.string.isRequired,
+      buildVersion: PropTypes.string.isRequired,
+      userId: PropTypes.number.isRequired,
+      isAutoAnalyzerEnabled: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
+      isPatternAnalyzerEnabled: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]).isRequired,
+      projectInfoId: PropTypes.number.isRequired,
+      isAdmin: PropTypes.bool.isRequired,
+    }).isRequired,
```

Comment:
Please create a PropTypes shape and reuse it.
You can store it f.e. where we store the events.